CSharp/Library/Microsoft.Bot.Builder.Azure/AzureStorageQueueActivityLogger.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.History;
+using Microsoft.Bot.Connector;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Azure
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class AzureStorageQueueActivityLogger : IActivityLogger
```

Comment:
Consider: AzureQueueActivityLogger would be a valid and shorter name. Not mandatory at all, just a thought